src/components/Editor/EditorControlPane/EditorControlPane.css

```diff
@@ -57,7 +57,7 @@
   max-width: 800px;
   margin: 0 auto;
 
-  & p {
+  & p:not(.nc-controlPanel-hint) {
```

Comment:
@erquhart I have this mostly rebased, just not sure how to add this line. Any ideas?

Reply:
You can still use the `not` selector here because Emotion components can be used as selectors via string interpolation. Let's say you created a `ControlHint` styled component, you could target it like so:

```
  ...
  & p:not(${ControlHint}) {
    font-size: 16px;
  }
```

Under the hood, Emotion creates a unique classname that applies to every instance of a styled component, so the interpolation is replaced with the classname for the component.

Reply:
Updated, ty!